exercises/grep/description.md

```diff
@@ -1,77 +1,25 @@
-# Description
+# Instructions
```

Comment:
This really is a description file. The heading is stripped out when imported into a track. It should not be changed to `instructions`.

Reply:
Done.